KsApi/models/Update.swift

```diff
@@ -49,7 +49,7 @@ extension Update: Argo.Decodable {
       <*> json <| "sequence"
       <*> (json <| "title" <|> .success(""))
     return tmp3
-      <*> json <| "urls"
+      <*> ((json <| "urls" >>- tryDecodable) as Decoded<Update.UrlsEnvelope>)
```

Comment:
Should this logic in this line go in here or directly in the decode method of the `Update.UrlsEnvelope` extension? Not quite sure.

Reply:
I would say it should stay here since eventually Argo will be removed and i don't see reasons for `Update.UrlsEnvelope` to know about Swift - to - Argo conversion process